cmd/update-index/main.go

```diff
@@ -130,8 +130,14 @@ func (b Binary) Row() string {
 	<td>%s</td>
 	<td>%s</td>
 	<td>%s</td>
-    <td><span title="download"><a href="https://%s">%s</a></span></td>
-    <td><span class="icon"><i class="fa fa-copy"></i></span><span title="copy to clipboard"><a class="copy" href="https://%s">  %s</a></span></td>`, b.Version, b.OperatingSystem, b.Architecture, b.downloadLink(), b.Name, b.downloadLink(), b.downloadLink())
+	<td><span title="download"><a href="https://%s">%s</a></span></td>
+	<td>
+		<span class="icon"><i class="fa fa-copy"></i></span>
+		<span title="copy to clipboard">
+			<a class="copy" href="https://%s">  %s</a>
+			(<a href="https://%s.sha256">checksum</a>)
+		</span>
+	</td>`, b.Version, b.OperatingSystem, b.Architecture, b.downloadLink(), b.Name, b.downloadLink(), b.downloadLink(), b.downloadLink())
```

Comment:
Maybe we can put this into a template as follow-up. Would it make easier to read from my perspective :+1:

Reply:
I agree, when I saw it I had the similar idea too. Will open another PR after this.

Reply:
Opened #18 to address the template suggestion.

Reply:
Rebased this over #18 which just got merged.